mediagrains/grains/EventGrain.py

```diff
@@ -175,7 +175,10 @@ def __init__(self,
         if 'data' not in self.meta['grain']['event_payload']:
             self.meta['grain']['event_payload']['data'] = []
 
-    @property
+    # ignoring typing here because mypy does not accept narrowing of the type here
```

Comment:
I wonder if a passing the output via a typeguard function that verified the output was indeed `bytes` would satisfy the typechecker?

Reply:
I might be mistaken, but I don't think that would work in this case because mypy has an issue with the declaration of the return type: `error: Signature of "data" incompatible with supertype "Grain"  [override]`. I tried adding an assert in the method that checks the type using a type guard and it still complains about the signature.

mypy doesn't detect that `bytes` is compatible with / narrowing of `Union[SupportsBytes, bytes, ndarray]`.

Reply:
Oh I see. That does seem like a problem with the typechecker. I can approve this now.